### PR TITLE
Splitting the `Bridge` tests

### PR DESCRIPTION
**Pull request by lukasz-zimnoch**
Closes: #173 
Depends on: https://github.com/keep-network/tbtc-v2/pull/266

This pull request splits the `Bridge.test.ts` file into specialized test files. This should give a better structure for `Bridge` tests and should make them easier to reason about.

This PR also incorporates changes to the former `Bridge.test.ts` introduced by https://github.com/keep-network/tbtc-v2/pull/209

### CODE REVIEW

**Comment by lukasz-zimnoch** [comment]
Ready for full review. Will undraft once #266 lands on main.

File: solidity/test/bridge/Bridge.Deposit.test.ts [hunk 1]
```diff
@@ -0,0 +1,2745 @@
...
+                      "0x28e081f285138ccbe389c1eb8985716230129f89",
+                      "0x60bcea61",
+                      reveal.vault
+                    )
+                })
+              })
+
+              context("when deposit is not routed to a vault", () => {
+                let tx: ContractTransaction
+                let nonRoutedReveal
```
Comment (tomaszslabon) [inline solidity/test/bridge/Bridge.Deposit.test.ts:197]: Would setting a type make sense here?
```
let nonRoutedReveal: DepositRevealInfoStruct
```

Imported from 
```
import { DepositRevealInfoStruct } from "../../typechain/Bridge"
```

It occurs multiple times in the files.

**Comment by tomaszslabon** [comment]
Besides [this comment](https://github.com/keep-network/tbtc-v2/pull/270#discussion_r869019759) I do not have any review comments.

File: solidity/test/bridge/Bridge.Deposit.test.ts (hunk 1 above)
Comment (lukasz-zimnoch) [reply solidity/test/bridge/Bridge.Deposit.test.ts]: See https://github.com/keep-network/tbtc-v2/pull/270/commits/058ce527345ff8e3bf710a025997203f830c14b6

Comment (tomaszslabon) [reply solidity/test/bridge/Bridge.Deposit.test.ts]: There is still one more left:
```
let nonTrustedVaultReveal
```

**Comment by pdyraga** [comment]
👀

**Review by pdyraga — APPROVED** [review]

**Pull request merged**